examples/devconsole_v1alpha1_gitsource_cr.yaml

```diff
@@ -9,6 +9,4 @@ spec:
   httpProxy: http://proxy.example.com
   httpsProxy: https://proxy.example.com
   noProxy: somedomain.com, otherdomain.com
-  secretRef:
```

Comment:
Why are these two lines removed?

Reply:
@akashshinde I think we should have 2 example files: one with secret named `devconsole_v1alpha1_gitsource_secret_cr.yaml` and this one (without secret). In my Pr I also removed it as it's not convenient when testing public repo.

Reply:
@corinnekrych Yeah that's exactly what I was also thinking, I should also include `Secret` template for reference.